WikiWatershed/model-my-watershed
# Reduce file system permissions for environment variable files

## Overview

Narrow the file system permissions for environment variable files so that only the read bit is given to user and group. In addition, give the directory that houses the environment variable files read and execute.

Connects #2529 

## Testing Instructions

Re-provision the `app` and `tiler` virtual machines; ensure the application is still functional afterwards.

For testing the `cloud-config` specific changes, they were packaged up in AMIs and deployed to staging yesterday in:

- http://civicci01.internal.azavea.com/view/mmw/job/model-my-watershed-packer-app-and-worker/1012/
- http://civicci01.internal.azavea.com/view/mmw/job/model-my-watershed-packer-tiler/929/
